src/components/ErrorLoadPage/ErrorLoadPage.tsx

```diff
@@ -0,0 +1,7 @@
+export const ErrorLoadPage = () => {
+  return (
+    <p data-cy="peopleLoadingError" className="has-text-danger">
+      Something went wrong
+    </p>
+  );
+};
```

Comment:
This doesn't need to be a separate page, it is just error message to show instead of people table.